### PR TITLE
Add docs for explore page tabs

### PR DESCRIPTION
**Pull request by ketankartoza**
This PR is for the issue #1682 

@Jeremy-Prior, please refrain from reviewing this PR as it is dependent on the completion of PR #1641. Once PR #1641 is merged, I will rebase this PR and you for review.

### CODE REVIEW

**Comment by ketankartoza** [comment]
@Jeremy-Prior This PR is ready for review now. Thanks

File: docs/src/user/manual/explore/filters.md [hunk 1]
```diff
@@ -54,72 +59,263 @@ The filter side panel enhances the user experience by allowing them to focus on
...
+* **Permission:** The `can view property filter` permission must be added to the group on the admin page,
 to allow the user group (e.g. `National data scientist`) to see the property filter.
 
-![Property filter](./img/filters-6.png)
+    ![Property filter](./img/filters-6.png)
 
-1. **Select All:** `Select All` of the properties.
-2. **Show selected property count:** A user will see the count of the property selected.
-3. **Cross Icon**: Click on the `Cross` icon to clear the property filter.
+1. **Property Filter:** In the property filter, all properties are initially selected as the default setting. Users have the freedom to handpick a property of their choice. If a user is associated with multiple organisations, the filter intelligently displays a list of properties aligned with the selected organisation, ensuring a tailored and organisation-specific selection experience.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/explore/filters.md:91]:
```suggestion
1. **Property Filter:** In the property filter, all properties are initially selected as the default setting. Users have the freedom to hand-pick a property of their choice. If a user is associated with multiple organisations, the filter intelligently displays a list of properties aligned with the selected organisation, ensuring a tailored and organisation-specific selection experience.
```

File: docs/src/user/manual/explore/filters.md [hunk 2]
```diff
@@ -54,72 +59,263 @@ The filter side panel enhances the user experience by allowing them to focus on
...
+
+#### 5. Property Filter
+
+* **Filter Description:** The property filter helps users filter data based on properties. This filter is dependent on the organisation filter.
+It means the options will be based on the selected organisations.
+* **Usage:** Select a property from the available options to view data related to that property.
+
+    ![Property filter](./img/filters-14.png)
+
+1. **Property Filter:** In the property filter, all properties are initially selected as the default setting. Users have the freedom to handpick a property of their choice. If a user is associated with multiple organisations, the filter intelligently displays a list of properties aligned with the selected organisation, ensuring a tailored and organisation-specific selection experience.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/explore/filters.md:179]:
```suggestion
1. **Property Filter:** In the property filter, all properties are initially selected as the default setting. Users have the freedom to hand-pick a property of their choice. If a user is associated with multiple organisations, the filter intelligently displays a list of properties aligned with the selected organisation, ensuring a tailored and organisation-specific selection experience.
```

File: docs/src/user/manual/explore/filters.md [hunk 3]
```diff
@@ -54,72 +59,263 @@ The filter side panel enhances the user experience by allowing them to focus on
...
+
+#### 4. Property Filter
+
+* **Filter Description:** The property filter helps users filter data based on properties. This filter is dependent on the organisation filter.
+It means the options will be based on the selected organisations.
+* **Usage:** Select a property from the available options to view data related to that property.
+
+    ![Property filter](./img/filters-14.png)
+
+1. **Property Filter:** In the property filter, all properties are initially selected as the default setting. Users have the freedom to handpick a property of their choice. If a user is associated with multiple organisations, the filter intelligently displays a list of properties aligned with the selected organisation, ensuring a tailored and organisation-specific selection experience.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/explore/filters.md:255]:
```suggestion
1. **Property Filter:** In the property filter, all properties are initially selected as the default setting. Users have the freedom to hand-pick a property of their choice. If a user is associated with multiple organisations, the filter intelligently displays a list of properties aligned with the selected organisation, ensuring a tailored and organisation-specific selection experience.
```

File: docs/src/user/manual/explore/filters.md [hunk 4]
```diff
@@ -54,72 +59,263 @@ The filter side panel enhances the user experience by allowing them to focus on
...
+
+#### 3. Property Filter
+
+* **Filter Description:** The property filter helps users filter data based on properties. This filter is dependent on the organisation filter.
+It means the options will be based on the selected organisations.
+* **Usage:** Select a property from the available options to view data related to that property.
+
+    ![Property filter](./img/filters-14.png)
+
+1. **Property Filter:** In the property filter, all properties are initially selected as the default setting. Users have the freedom to handpick a property of their choice. If a user is associated with multiple organisations, the filter intelligently displays a list of properties aligned with the selected organisation, ensuring a tailored and organisation-specific selection experience.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/explore/filters.md:321]:
```suggestion
1. **Property Filter:** In the property filter, all properties are initially selected as the default setting. Users have the freedom to hand-pick a property of their choice. If a user is associated with multiple organisations, the filter intelligently displays a list of properties aligned with the selected organisation, ensuring a tailored and organisation-specific selection experience.
```

File: docs/src/user/manual/explore/filters.md [hunk 5]
```diff
@@ -13,6 +13,9 @@ context_id: GD6YdjhPx43u8TKLi3eNNE
 ---
 
 # Filter Side Panel Documentation
+# Map Tab Filter Panel Overview
+
+> **Disclaimer:** Please read the following information carefully. The screenshots may vary slightly due to various user roles on the platform.
 
 The filter side panel allows users to refine and narrow down data based on specific criteria. There are different types of filters are available for customisation.
 The filter side panel enhances the user experience by allowing them to focus on specific subsets of data based on their preferences, making it easier to analyse and work with the dataset effectively.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/explore/filters.md:21]: There should only be one level 1 heading per file, and also maybe rearrange things like:
```suggestion
# Filter Side Panel Documentation

> **Disclaimer:** Please read the following information carefully. The screenshots may vary slightly due to various user roles on the platform.

The filter side panel allows users to refine and narrow down data based on specific criteria. There are different types of filters are available for customisation.

## Map Tab Filter Panel Overview

The filter side panel enhances the user experience by allowing them to focus on specific subsets of data based on their preferences, making it easier to analyse and work with the dataset effectively.
```

**Review by Jeremy-Prior — CHANGES_REQUESTED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 47 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 46 sec

**Review by Jeremy-Prior — APPROVED** [review]
LGTM

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 26 sec

**Pull request merged**